support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -332,20 +330,23 @@ export function ThreeTierLanding(): JSX.Element {
 
 	const useGenericCheckout = abParticipations.useGenericCheckout === 'variant';
 
+	const enableSingle = countryGroupId === UnitedStates; // TODO - use campaign config
```

Comment:
If we merge this in - it will become immediately available. Assuming this is what the `TODO` is about.

Could we run this as an a/b test pre campaigns?

Reply:
AB test is one way. I was hoping to use a [campaign switch](https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/helpers/campaigns/campaigns.ts#L32) instead. I'll branch off this branch and have a go at doing it this way first

Reply:
Here's a PR to use campaign settings - https://github.com/guardian/support-frontend/pull/6249